docs/INSTALL.md

```diff
@@ -96,6 +95,102 @@ required:
     ```
 
 
+## Configuration: AF_PACKET mode
+
+To configure/install the UPF in AF_PACKET mode, the following changes are required:
+
+1. Enable AF_PACKET mode in configuration file
+
+    ```patch
+    diff --git a/conf/upf.json b/conf/upf.json
+    index 37447b7..b56e1fd 100644
+    --- a/conf/upf.json
+    +++ b/conf/upf.json
+    @@ -1,10 +1,10 @@
+     {
+         "": "Vdev or sim support. Enable `\"mode\": \"af_xdp\"` to enable AF_XDP mode, or `\"mode\": \"af_packet\"` to enable AF_PACKET mode, `\"mode\": \"sim\"` to generate synthetic traffic from BESS's Source module or \"mode\": \"\" when running with UP4",
+         "": "mode: af_xdp",
+    -    "": "mode: af_packet",
+    +    "mode": "af_packet",
+         "": "mode: sim",
+         "": "mode: cndp",
+    -    "mode": "dpdk",
+    +    "": "mode: dpdk",
+
+         "table_sizes": {
+             "": "Example sizes based on sim mode and 50K sessions. Customize as per your control plane",
+    @@ -62,13 +62,13 @@
+
+         "": "Gateway interfaces",
+         "access": {
+    -        "ifname": "ens803f2",
+    +        "ifname": "ens801f0",
```

Comment:
we can keep existing interface name itself, anyway the user need to update this irrespective of modes.

Reply:
Well, the thing is that if you do not tell the user what to change (e.g., changes are needed in `upf.json`), the user (especially new users) will not know that this is also needed.